exchangelib/version.py

```diff
@@ -270,12 +278,13 @@ def _get_version_from_service(cls, protocol, api_version):
 
     @classmethod
     def from_response(cls, requested_api_version, response):
+        header = None
         try:
             header = to_xml(response.text, encoding=response.encoding).find('{%s}Header' % SOAPNS)
-            if not header:
+            if len(header) < 1:
```

Comment:
`len()` doesn't make much sense because `find()` returns either an XML element or `None`.

`if header is None` is a better alternative because XML elements may evaluate to `False`.